File: csharp/10_input.cs
using System;

class Program {
    static void Main() {
        Console.Write("What is your name? ");
        string name = Console.ReadLine();
        Console.WriteLine($"Hello {name}!");

        Console.Write("How old are you? ");
        int age = int.Parse(Console.ReadLine());
        Console.WriteLine($"You are {age} years old.");
        Console.WriteLine($"Next year you will be {age + 1}");
    }
}

File: rust/10_input.rs
use std::io;

fn main() {
    let mut name = String::new();
    println!("What is your name?");
    io::stdin().read_line(&mut name).expect("failed");
    let name = name.trim();
    println!("Hello {}!", name);

    let mut age = String::new();
    println!("How old are you?");
    io::stdin().read_line(&mut age).expect("failed");
    let age: i32 = age.trim().parse().expect("not a number");
    println!("You are {} years old.", age);
    println!("Next year you will be {}", age + 1);
}
